packages/code-du-travail-frontend/src/common/Answer.js

```diff
@@ -126,4 +126,7 @@ const ShareContainer = styled(Container)`
   @media (max-width: ${breakpoints.mobile}) {
     justify-content: flex-start;
   }
+  @media print {
```

Comment:
do you need to remove it since it's also hide in the `<Share Component /> ?

Reply:
In the two used, one has a text inside so I'd say yes. Plus it's safer for the print layout too